### DISCUSSION TITLE
Which forecaster models allow global forecasting?

### DISCUSSION THREAD

**Discussion by kdekker-kdr4** [alan-turing-institute/sktime#6832] [Q&A] [answered]
I know global forecasting is supported with regression based forecasting, when enabling the pooling option "global" in the reduction for the regression model. Are there also other forecasters that can do global forecasting?

**Comment by fkiraly** [comment, marked as answer]
There is a new tag from the next release on, `capability:global_forecasting`.

The global forecasting interface is also subject to consolidation, and "global" capabilities are added for existing forecasters that should support it.

@Xinyu-Wu-0000 and @benHeid may also like to add sth.

**Comment by XinyuWuu** [comment]
Currently only pytorch-forecasting models can do global forecasting. Also check [huggingface models](https://github.com/sktime/sktime/pull/6686) and  [neuralforecast models](https://github.com/sktime/sktime/pull/6666). Also two more pytorch-forecasting models will be added through https://github.com/sktime/sktime/pull/6551.
